tutorials/scripting/scene_unique_nodes.rst

```diff
@@ -40,3 +40,71 @@ name in the path for ``get_node()``. For example:
  .. code-tab:: csharp
 
     GetNode<Button>("%RedButton").Text = "Hello";
+
+Same-scene limitation
+---------------------
+
+A scene unique node can only be retrieved by a node inside the same scene.
```

Comment:
```suggestion
A scene unique node can only be retrieved by a node inside of the same scene.
```

Reply:
That doesn't sound right to me (AmE). I don't know enough grammar terminology to say exactly why, though. 😄 A couple online grammar checkers say the `of` is not necessary if I paste in this sentence.